spinalcordtoolbox/reports/qc.py

```diff
@@ -419,15 +522,25 @@ def __init__(self, input_file, command, args, orientation, dest_folder, dpi=300,
         self.root_folder = dest_folder
         self.mod_date = datetime.datetime.strftime(datetime.datetime.now(), '%Y_%m_%d_%H%M%S.%f')
         self.qc_results = os.path.join(dest_folder, '_json/qc_' + self.mod_date + '.json')
-        self.bkg_img_path = os.path.join(dataset, subject, contrast, command, self.mod_date, 'bkg_img.png')
-        self.overlay_img_path = os.path.join(dataset, subject, contrast, command, self.mod_date, 'overlay_img.png')
+        if command in ['sct_fmri_moco', 'sct_dmri_moco']:
+            self.bkg_img_path = os.path.join(dataset, subject, contrast, command, self.mod_date, 'bkg_img.gif')
+            self.overlay_img_path = os.path.join(dataset, subject, contrast, command, self.mod_date, 'overlay_img.gif')
+        else:
+            self.bkg_img_path = os.path.join(dataset, subject, contrast, command, self.mod_date, 'bkg_img.png')
+            self.overlay_img_path = os.path.join(dataset, subject, contrast, command, self.mod_date, 'overlay_img.png')
```

Comment:
Nitpick: This could be simplified a little:

```python
    if command in ['sct_fmri_moco', 'sct_dmri_moco']:
        ext = "gif"
    else: 
        ext = "png"
    self.bkg_img_path = os.path.join(dataset, subject, contrast, command, self.mod_date, f"bkg_img.{ext}")
    self.overlay_img_path = os.path.join(dataset, subject, contrast, command, self.mod_date, f"overlay_img.{ext}")
```